interface/forms/eye_mag/SpectacleRx.php

```diff
@@ -444,239 +410,242 @@ function delete_me(delete_id) {
-                                                <tr class="bold underline">
-                                                    <td></td>
-                                                    <td><?php echo xlt('Sph{{Sphere}}'); ?></td>
-                                                    <td><?php echo xlt('Cyl{{Cylinder}}'); ?></td>
-                                                    <td><?php echo xlt('Axis{{Axis in a glasses prescription}}'); ?></td>
-                                                    <td><?php echo xlt('BC{{Base Curve}}'); ?></td>
-                                                    <td><?php echo xlt('Diam{{Diameter}}'); ?></td>
-                                                    <?php
-                                                    if ($adds) {
-                                                        ?>
-                                                            <td><?php echo xlt('ADD'); ?></td>
-                                                        <?php }
-                                                    ?>
-                                                </tr>
-                                                <tr>
-                                                    <td></td>
-                                                    <td><?php echo text($row['OSSPH']); ?></td>
-                                                    <td><?php echo text($row['OSCYL']); ?></td>
-                                                    <td><?php echo text($row['OSAXIS']); ?></td>
-                                                    <td><?php echo text($row['OSBC']); ?></td>
-                                                    <td><?php echo text($row['OSDIAM']); ?></td>
-                                                    <?php
-                                                    if ($adds) {
-                                                        ?>
-                                                            <td><?php echo text($row['OSADD']); ?></td>
+                                                        if ($adds) {
+                                                            ?>
+                                                                <td><?php echo text($row['ODADD']); ?></td>
+                                                            <?php } ?>
+                                                    </tr>
+                                                    <tr>
+                                                        <td colspan="2" class="right font-weight-bold">
+                                                            <?php echo xlt('Brand'); ?>: <br />
+                                                            <?php echo xlt('Quantity'); ?>: <br />
+                                                            <?php echo xlt('Supplier'); ?>: </td>
+                                                        <td colspan="5" class="left" style="padding-left:10px;vertical-align:top;">
```

Comment:
Replace vertical align top with BS4 class and in this entire file replace `left` class with `text-left` BS4 class